src/app/(nextjs_migration)/(authenticated)/admin/feature-flags/page.tsx

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
@Vinnl was advising me to use the `(proper_react)` subdir for new code, should that apply here?

Reply:
Yes please! I'd like to wholesale remove `(nextjs_migration)` once we release the new version, so anything that should live on beyond that would ideally not be in that directory.

Reply:
Moved to (proper_react).. At this point, we can't see that what's in that folder from the frontend? @rhelmer @Vinnl

Reply:
According to the current file path, the URL to the page should be `/redesign/admin/feature-flags/` - is that what you mean?

Reply:
Yes thanks!